Documentation/ApiOverview/SymfonyExpressionLanguage/Index.rst

```diff
@@ -47,13 +47,15 @@ The provider is registered in the extension file :file:`/Configuration/Expressio
 the extension's custom PHP class name:
```

Comment:
```suggestion
the extension's custom PHP class name::
```

Reply:
This will trigger the following block to be formatted as code instead of quote. See "shorthand" here:

https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/Codeblocks.html

Notice that the following "Implement provider within extension" section needs the same fix.